### ISSUE TITLE
Footnotes Not Rendering in PDF (but work in HTML)

### ISSUE THREAD

**Issue by tajmone** [asciidoctor/asciidoctor-pdf#927]
This is somehow related to Issue #85, but not the same.

I have encountered a problem when converting to PDF a Table which contains footnotes. Here is the Table source ([adoc source link]):

```asciidoc
[[table-of-options]]
.Adventure Settings via OPTION
[cols="<20m,<40d,<40d",options="header"]
|===============================================================================
| Option name | Possible values | Default value

| Language
| `English`, `Swedish`, ``German``footnote:[Other non-English languages may be supported in the future depending on demand.]
| `English`

| Width
| 24-255
| 80footnote:[Width and Length is overridden by the actual terminal or window size, if available.]
```

The problem is with "` ``German``footnote:[`" and "`80footnote:[`", which are correctly rendered as footnotes in the HTML backend, but when using Asciidoctor PDF they are not rendered as notes, but the contents of the note are rendered as inline text (withing brackets and preceded by a space):

```
Language English, Swedish, German [1: Other non-
English languages may be supported
in the future depending on demand.]
```

and

```
Width 24-255 80 [2: Width and Length is overridden
by the actual terminal or window size,
if available.]
```

The only workaround I can come up with is adding a separating space before `footnote[`, but then that space would show up in the final documents as well (which doesn't look good). In the Documentation examples, no spaces are inserted between the footnote and the word it refers to, but in all examples there is a punctuation mark between them. In real life scenarios, footnotes might be attached directly to a word which is not followed by punctuation, so probably any spaces before the `footnote[` macro should be removed in the final rendered document.

The HTML backend handles well the footnotes in the above table, so probably this is a bug in the Asciidoctor PDF backend. Here is a live HTML preview of how the above table is rendered via HTML backend:

- http://htmlpreview.github.io/?https://github.com/alan-if/alan-docs/blob/8af2b68/manual/manual.html#table-of-options

Any ideas on a temporary workaround for the problem? (possibly, that doesn't introduce a space between the word and the note number)

[adoc source link]: https://github.com/alan-if/alan-docs/blame/8af2b68/manual/manual_03.asciidoc#L119

**Comment by tajmone** [comment]
This issue is related to #73.

The problem doesn't seem to be the lack of space (or punctuation) before the `footnote:[`, but how Asciidoctor PDF handles footnotes in Tables.

If I set the columns to `a`, the notes are rendered in the PDF, but inside the cell (like @Gitward mentions in #73).

**Comment by mojavelinux** [comment]
This was fixed somewhere along the way, perhaps when #85 got fixed. I've submitted a test to verify that behavior.

**Comment by tajmone** [comment]
Great to see this fixed! thanks.
